### ISSUE TITLE
Debug output is replicated in stdout

### ISSUE THREAD

**Issue by ekquasar** [cisco-open/fsoc#103]
Running:
```bash
fsoc config set --auth oauth --server REDACTED -o json
```

gives this output:
```bash
The --server option is now deprecated. In the future, please use --url instead. We will set the url to "REDACTED" for you now
```
^ this error message is replicated in both stderr and stdout. This is a problem for scripting applications that use stdout programmatically. Debugging and error output should be only printed to stderr.

**Comment by pnickolov** [comment]
@ekquasar  will check and try to resolve. The `config set` command, afaik, doesn't have any data output (so a workaround would be to send stdout to /dev/null) but I agree this can/should be fixed.

**Comment by ekquasar** [comment]
Yeah that's the workaround I ended up with, but as far as "polish" is concerned, this stuck out to me as a "sharp edge" so I figured I'd bring it to maintainers' attention.
